### DISCUSSION TITLE
Version Compare

### DISCUSSION THREAD

**Discussion by Swing366** [iTwin/itwinjs-core#1638] [Q&A] [answered]
Hello,guys

I created three files on MS, the Version_2 file based on Version_1, added two balls, the version_3 based on version_2, add two balls too. then respectively importing them and creating three named version:1/2/3.

Questions are:

1. Imported the file that created on MS, whey using Civil bridge? See screenshot Civilbridge.jpg
2. 3 version compare 1, the cone turned green color, is it right? I think the cone should be grey or black one. See the video:Version Compare.MP4.
The dataset in the below:
[Pictures.zip](https://github.com/imodeljs/imodeljs/files/6660219/Pictures.zip)
Any Suggestion, I will appreciate.

**Comment by pmconne** [comment, marked as answer]
The timeline of an iModel is not produced from **copies** of source files; it is produced from **changes** to those files over time. The following would produce the results you seem to be expecting:
- Create .dgn file containing a red cylinder.
- Import it and create named version Version_1.
- In the same .dgn file, add 2 spheres to the model.
- Rerun the bridge and create named version Version_2.
- In the .dgn file, add 2 more spheres.
- Rerun the bridge and create named version Version_3.

This will produce one model containing 5 elements, plus 3 named versions showing how the contents of that model changed over time.

**Reply by Swing366** [reply]
Thanks for kindly reply. Got it.
